CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ description: |-
 - Add `cpu_generation` param to terraform schema of `vkcs_compute_flavor`
 - improve search flavor using min_ram and min_disk
 - Improve error messages for `vkcs_compute_flavor`
+- Add `security_group_ids` param to `vkcs_compute_instance` resource
```

Comment:
argument, no quotas
new arguments should be above improvements

Reply:
done